aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -248,6 +251,25 @@ public class AwsProperties implements Serializable {
    */
   public static final String S3FILEIO_SESSION_TOKEN = "s3.session-token";
 
+  /**
+   * Configure the AWS credentials provider used to create AWS clients. A fully qualified concrete
```

Comment:
The last two sentences here seem similar, do we need both?

Reply:
Fixed